ui/components/src/EChart/EChart.stories.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { EChart } from '@perses-dev/components';
+import { use } from 'echarts/core';
+import { LineChart as EChartsLineChart } from 'echarts/charts';
+import { CanvasRenderer, SVGRenderer } from 'echarts/renderers';
+import {
+  GridComponent,
+  DataZoomComponent,
+  MarkAreaComponent,
+  MarkLineComponent,
+  MarkPointComponent,
+  TitleComponent,
+  ToolboxComponent,
+  TooltipComponent,
+  LegendComponent,
+} from 'echarts/components';
+import { action } from '@storybook/addon-actions';
+
+use([
+  EChartsLineChart,
+  GridComponent,
+  DataZoomComponent,
+  MarkAreaComponent,
+  MarkLineComponent,
+  MarkPointComponent,
+  TitleComponent,
+  ToolboxComponent,
+  TooltipComponent,
+  LegendComponent,
+  CanvasRenderer,
+  SVGRenderer,
+]);
+
+const meta: Meta<typeof EChart> = {
+  component: EChart.type,
```

Comment:
I peeked in the LineChart stories file and saw the `component` pointed to the actual component, should this do that too?

Reply:
Hmm, I _think_ it should, but there _might_ be an edge case reason why I did this. I was mostly copy-pasting from the investigation I did in https://github.com/perses/perses/pull/984, and I vaaaaguely recall this maybe being needed in cases where we use `forwardRef`. Lemme poke at it and either fix it to the normal way or clarify why it's doing it this way.

Reply:
It works fine without the `.type`, so I removed it. I may have hit something funny with the memo wrapper before that's since been fixed.